tests/cppunit/disk_test.cc

```diff
@@ -0,0 +1,202 @@
+  key_ = "listdisk_key";
+  values_.resize(2);
+  std::vector<int>value_size{1,1024};
+  for(int i = 0;i < int(values_.size()); i++){
+    values_[i] = std::string(value_size[i],'a');
+  }
+  int ret = 0;
+  uint64_t sum = 0;
+  for (int i = 0; i < int(values_.size()); i++) {
+    std::vector<Slice>tmpv{values_[i]};
+    sum += values_[i].size();
+    rocksdb::Status s = list->Push(key_, tmpv, false, &ret);
+    EXPECT_TRUE(s.ok() && ret == i + 1);
+    // waiting for data write
+    std::this_thread::sleep_for(std::chrono::milliseconds(100));
+    uint64_t key_size;
+    EXPECT_TRUE(disk->GetKeySize(key_, kRedisList, &key_size).ok());
+    EXPECT_GE(key_size, sum);
+  }
+  list->Del(key_);
+}
+
+TEST_F(RedisDiskTest, ZsetDisk) {
+  std::unique_ptr<Redis::ZSet> zset = Util::MakeUnique<Redis::ZSet>(storage_, "disk_ns_zet");
+  std::unique_ptr<Redis::Disk> disk = Util::MakeUnique<Redis::Disk>(storage_, "disk_ns_zet");
+  key_ = "zsetdisk_key";
+  std::vector<MemberScore> mscores(2);
+  std::vector<int>value_size{1,1024};
+  for(int i = 0;i < int(value_size.size()); i++){
+    mscores[i].member = std::string(value_size[i],'a');
+    mscores[i].score = 1.0 * value_size[int(values_.size()) - i - 1];
+  }
+  int ret = 0;
+  uint64_t sum = 0;
+  for (int i = 0; i < int(mscores.size()); i++) {
+    std::vector<MemberScore> tmpv{mscores[i]};
+    sum += 2 * mscores[i].member.size();
+    rocksdb::Status s = zset->Add(key_, 0, &tmpv, &ret);
+    EXPECT_TRUE(s.ok() && ret == 1);
+    // waiting for data write
+    std::this_thread::sleep_for(std::chrono::milliseconds(100));
+    uint64_t key_size;
+    EXPECT_TRUE(disk->GetKeySize(key_, kRedisZSet, &key_size).ok());
+    EXPECT_GE(key_size, sum);
+  }
+  zset->Del(key_);
+}
+
+TEST_F(RedisDiskTest, BitmapDisk) {
+  std::unique_ptr<Redis::Bitmap> bitmap = Util::MakeUnique<Redis::Bitmap>(storage_, "disk_ns_bitmap");
+  std::unique_ptr<Redis::Disk> disk = Util::MakeUnique<Redis::Disk>(storage_, "disk_ns_bitmap");
+  key_ = "bitmapdisk_key";
+  uint32_t offsets[] = {0, 123, 1024*8, 1024*8+1, 3*1024*8,  3*1024*8+1};
+
+  for (int i = 0; i < 6 ;i++) {
+    bool bit = false;
+    bitmap->GetBit(key_, offsets[i], &bit);
+    EXPECT_FALSE(bit);
+    bitmap->SetBit(key_, offsets[i], true, &bit);
+    bitmap->GetBit(key_, offsets[i], &bit);
+    EXPECT_TRUE(bit);
+    // waiting for data write
+    std::this_thread::sleep_for(std::chrono::milliseconds(100));
+    uint64_t key_size;
+    EXPECT_TRUE(disk->GetKeySize(key_, kRedisBitmap, &key_size).ok());
+    EXPECT_GE(key_size, i + 1);
+  }
+  bitmap->Del(key_);
+}
+
+TEST_F(RedisDiskTest, SortedintDisk) {
+  std::unique_ptr<Redis::Sortedint> sortedint = Util::MakeUnique<Redis::Sortedint>(storage_, "disk_ns_sortedint");
+  std::unique_ptr<Redis::Disk> disk = Util::MakeUnique<Redis::Disk>(storage_, "disk_ns_sortedint");
+  key_ = "sortedintdisk_key";
+  int ret;
+  for(int i=0;i<10;i++){
```

Comment:
Just formatting (add spaces like on line 174).

Reply:
I noticed that the running results of ci are somewhat different from those of my native machine. At the same time, because this is a draft pr, I am using ci for testing.I found that a byte of data is often not detected due to the problem with the settings of Writebatch(sync is default false).